nixos/modules/services/mail/mailman.nix

```diff
@@ -38,11 +38,11 @@ let
   webSettingsJSON = pkgs.writeText "settings.json" (builtins.toJSON webSettings);
 
   # TODO: Should this be RFC42-ised so that users can set additional options without modifying the module?
-  mtaConfig = pkgs.writeText "mailman-postfix.cfg" ''
+  mtaConfig = if cfg.enablePostfix then pkgs.writeText "mailman-postfix.cfg" ''
```

Comment:
This doesn't need to be conditional, since it will only be used if enablePostfix is set anyway. Maybe rename it to `postfixMtaConfig` for clarity?

Reply:
Good idea.